README.md

```diff
@@ -17,10 +17,15 @@ option.
 
 ### Compute
 
+Computes coverage from `target/catalog.json` and `target/manifest.json` files
+found in a dbt project. You can switch between docs or test coverage by using
+the option `--cov-type (doc|test)`.
+
 ```
 $ cd my-dbt-project
 $ dbt docs generate  # Generate catalog.json and manifest.json
-$ dbt-coverage compute  # Create coverage.json file and print coverage
+$ dbt-coverage compute  # Create coverage.json file with docs coverage (default) and print coverage
```

Comment:
@sweco I am contemplating which of these would be better as a default. It almost feels like two separate commands 🙂

Reply:
This is a good suggestion. What do you think about making the argument required and positional?

```
dbt-coverage compute (docs|test)
```

Reply:
@sweco That would indeed be quite nice!